app/helpers/work_packages_helper.rb

```diff
@@ -565,22 +565,29 @@ def work_package_form_category_attribute(form, _work_package, locals = {})
     return if locals[:project].categories.empty?
 
     category_field = work_package_form_field do
-      field = form.select(:category_id,
-                          (locals[:project].categories.map { |c| [c.name, c.id] }),
-                          include_blank: true)
-
-      if authorize_for('categories', 'new')
-
-        field += prompt_to_remote(icon_wrapper('icon icon-add icon4',
-                                               I18n.t(:label_work_package_category_new)),
-                                  I18n.t(:label_work_package_category_new),
-                                  'category[name]',
-                                  project_categories_path(locals[:project]),
-                                  class: 'no-decoration-on-hover grid-block',
-                                  title: I18n.t(:label_work_package_category_new))
+      label = form.label(:category_id)
+
+      field = content_tag(:span, class: "form--field-container") do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.